br/pkg/lightning/mydump/loader.go

```diff
@@ -99,6 +102,76 @@ func (m *MDTableMeta) GetSchema(ctx context.Context, store storage.ExternalStora
 	return string(schema), nil
 }
 
+// MDLoaderSetupConfig stores the configs when setting up a MDLoader.
+type MDLoaderSetupConfig struct {
+	MaxScanFiles int
+}
+
+// DefaultMDLoaderSetupConfig generates a default MDLoaderSetupConfig.
+func DefaultMDLoaderSetupConfig() *MDLoaderSetupConfig {
+	return &MDLoaderSetupConfig{
+		MaxScanFiles: 0,
```

Comment:
`MaxScanFiles = 0` is confused. Can you add some comment here to explain the why set it to 0.